mmv1/third_party/terraform/services/dataprocmetastore/resource_dataproc_metastore_service_test.go.erb

```diff
@@ -107,6 +107,34 @@ func TestAccDataprocMetastoreService_PrivateServiceConnect(t *testing.T) {
 	})
 }
 
+func TestAccMetastoreService_tags(t *testing.T) {
+	t.Skip()
+
+	t.Parallel()
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+	tagKey := acctest.BootstrapSharedTestTagKey(t, "metastore-services-tagkey")
+	tagValue := acctest.BootstrapSharedTestTagValue(t, "metastore-services-tagvalue", tagKey)
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckDataprocMetastoreServiceDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMetastoreServiceTags(context, map[string]string{tagKey: tagValue}),
+			},
+			{
+				ResourceName:            "google_dataproc_metastore_service.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"tags"},
```

Comment:
This probably fails right now as the field service_id, location, labels and terraform_labels are also not read back. So they should also be included in this list for it to pass. Please ensure the test passes.

Reply:
Sure